jobspec/test-fixtures/tg-network.hcl

```diff
@@ -1,12 +1,15 @@
 job "foo" {
   datacenters = ["dc1"]
+
```

Comment:
I used `hclfmt` to clean up my own changes and it turns out this file was dirty. Apologies for the review noise.